package.json

```diff
@@ -155,8 +156,11 @@
     "mocha": "^6.2.2",
     "mocha-junit-reporter": "^1.23.1",
     "nock": "^11.4.0",
+    "node-sass": "^4.13.1",
+    "nodemon": "^2.0.2",
```

Comment:
adding nodemon as a dev dependency as the scripts require it (and now node-sass is required as not pulled in as part of the sass middleware)